modelling/src/main/java/org/axonframework/modelling/command/AnnotationCommandTargetResolver.java

```diff
@@ -110,4 +133,76 @@ private Long asLong(Object fieldValue) {
             return Long.parseLong(fieldValue.toString());
         }
     }
-}
+
+    @Override
+    public String toString() {
+        return "AnnotationCommandTargetResolver [identifierAnnotation="
+                + identifierAnnotation + ", versionAnnotation="
+                + versionAnnotation + "]";
+    }
+
+    /**
+     * Builder class to instantiate a {@link AnnotationCommandTargetResolver}.
+     * <p>
+     * The TargetAggregateIdentifierAnnotation is defaulted to {@link TargetAggregateIdentifier}, 
+     * TargetAggregateVersionAnnotation to {@link TargetAggregateVersion}.
+     * 
+     * @author JohT
+     */
+    public static final class Builder {
+
+        private Class<? extends Annotation> identifierAnnotation = TargetAggregateIdentifier.class;
+        private Class<? extends Annotation> versionAnnotation = TargetAggregateVersion.class;
+
+        /**
+         * Sets the annotation, that marks the target aggregate identifier.
+         * <p>
+         * Defaults to {@link TargetAggregateIdentifier}.<br>
+         * <p>
+         * If you do not want your messages-module (as inner bounded context
+         * "API") to be dependent of axon annotations (e.g. to aim empty pom
+         * dependencies), then you can write your own annotation based on the
+         * {@link TargetAggregateIdentifier} without referencing the original
+         * one (as meta-annotation).
+         * 
+         * @param annotation
+         *            - {@link Class} of type {@link Annotation}.
+         * @return {@link Builder}
+         */
+        public Builder setTargetAggregateIdentifierAnnotation(Class<? extends Annotation> annotation) {
+            assertNonNull(annotation, "TargetAggregateIdentifierAnnotation may not be null");
+            this.identifierAnnotation = annotation;
+            return this;
+        }
+
+        /**
+         * Sets the annotation, that marks the target aggregate version.
+         * <p>
+         * Defaults to {@link TargetAggregateVersion}.
+         * <p>
+         * If you do not want your messages-module (as inner bounded context
+         * "API") to be dependent of axon annotations (e.g. to aim empty pom
+         * dependencies), then you can write your own annotation based on the
+         * {@link TargetAggregateVersion} without referencing the original one
+         * (as meta-annotation).
+         * 
+         * @param annotation
```

Comment:
Would you mind removing the - from this line here? That way it'll resemble how we typically write the @param javadoc lines in the framework.